src/components/EndorsementChain/EndorsementChainLayout/EndorsementChainError/EndorsementChainError.tsx

```diff
@@ -13,9 +13,7 @@ export const EndorsementChainErrorUnstyled: FunctionComponent<EndorsementChainEr
     <div className={`${className}`}>
       <div className="error-container">
         <div className="fixed-error-row">
-          <SvgIcon className="alert-icon">
-            <SvgIconAlert />
-          </SvgIcon>
+          <AlertTriangle className="alert-icon mr-2" color={`${vars.red}`} />
```

Comment:
alert-icon should change it to red right? how come still need color over here?

Reply:
oh previously the red was defined here https://github.com/TradeTrust/tradetrust-website/pull/239/files#diff-594bcced09275eff5d2ff48c4e563b13L29

now that the scoped css class is removed together with `SvgIcon`, will be using what `react-feather` provide instead, which is `color` prop

Reply:
ahhh ok then should remove the className?